### PR TITLE
[DEV-3555/3654] Update award v2 endpoint to include NAICS/PSC hierarchies

### PR DESCRIPTION
**Pull request by emily-brents**
**Description:**
Adds two fields, `psc_hierarchy` and `naics_hierarchy` to the `/v2/awards`endpoint response for contracts only.

**Technical details:**
Both the PSC and NAICS hierarchies use toptier_code, midtier_code, and base_code to return the level of the codes. The PSC codes include a subtier_code that is only used for R&D related codes (start with `A`). "Goods" PSCs will only return midtier and base codes

**Requirements for PR merge:**

1. [X] Unit & integration tests updated
2. [X] API documentation updated
3. [ ] Necessary PR reviewers:
    - [ ] Backend
    - [ ] Frontend
4. [X] Matview impact assessment completed (N/A)
5. [X] Frontend impact assessment completed
6. [X] Data validation completed
7. [X] Appropriate Operations ticket(s) created (N/A)
8. [X] Jira Ticket [DEV-3555](https://federal-spending-transparency.atlassian.net/browse/DEV-3555):
    - [X] Link to this Pull-Request
    - [X] Performance evaluation of affected (API | Script | Download)
    - [X] Before / After data comparison

**Area for explaining above N/A when needed:**
```
no change to matviews, no ops tickets needed
```
Request:/api/v2/awards/25881000
Response: 357ms
```
    "psc_hierarchy": {
        "base_code": {
            "description": "R&D- DEFENSE SYSTEM: AIRCRAFT (OPERATIONAL SYSTEMS DEVELOPMENT)",
            "code": "AC15"
        },
        "subtier_code": {
            "description": "R&D- DEFENSE SYSTEM: AIRCRAFT ",
            "code": "AC1"
        },
        "toptier_code": {
            "description": "RESEARCH AND DEVELOPMENT",
            "code": "A"
        },
        "midtier_code": {
            "description": "DEFENSE SYSTEMS R&D",
            "code": "AC"
        }
    },
    "naics_hierarchy": {
        "base_code": {
            "description": "Aircraft Manufacturing",
            "code": "336411"
        },
        "toptier_code": {
            "description": "Manufacturing",
            "code": "33"
        },
        "midtier_code": {
            "description": "Aerospace Product and Parts Manufacturing",
            "code": "3364"
        }
    },
```

### CODE REVIEW

File: usaspending_api/api_contracts/scripts/mock/drakov.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ const drakovOptions  = {
     sourceFiles: './contracts/**/*.md',
     serverPort: 5000,
     autoOptions: true,
-    delay: 2500 // 2.5 seconds
+    delay: 250 // 0.25 seconds
```
Comment (BrianZito) [inline usaspending_api/api_contracts/scripts/mock/drakov.js]: How come?

File: usaspending_api/awards/v2/data_layer/orm.py [hunk 2]
```diff
@@ -455,3 +459,70 @@ def fetch_transaction_obligated_amount_by_internal_award_id(internal_award_id: i
         return _sum["transaction_obligated_amount__sum"]
 
     return None
+
+
+def fetch_psc_hierarchy(psc_code: str) -> dict:
+    if psc_code[0].isalpha():
+        codes = [psc_code, psc_code[:2], psc_code[:1]]
+        if psc_code[0] == "A":
+            codes = [psc_code, psc_code[:2], psc_code[:1], psc_code[:3]]
```
Comment (BrianZito) [inline usaspending_api/awards/v2/data_layer/orm.py]: Can we have a comment explaining this logic? This is some obscure stuff...

File: usaspending_api/api_contracts/scripts/mock/drakov.js (hunk 1 above)
Comment (emily-brents) [reply usaspending_api/api_contracts/scripts/mock/drakov.js]: Ask @MaxwellKendall

File: usaspending_api/awards/v2/data_layer/orm.py (hunk 2 above)
Comment (BrianZito) [reply usaspending_api/awards/v2/data_layer/orm.py]: In retrospect, I was just confused by listing the codes twice. Could we do:
```
codes = [psc_code, psc_code[:2], psc_code[:1], psc_code[:3] if psc_code[0] == "A" else None] 
```
instead? We could then skip the second == "A" check and just see if codes[4] is None instead, and I think make things more clear

Comment (emily-brents) [reply usaspending_api/awards/v2/data_layer/orm.py]: sure

File: usaspending_api/settings.py [hunk 3]
```diff
@@ -34,7 +34,7 @@
 SECRET_KEY = get_random_string()
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = False
+DEBUG = True
```
Comment (tony-sappe) [inline usaspending_api/settings.py]: @emily-brents Need to remove

Comment (emily-brents) [reply usaspending_api/settings.py]: upsetting. just pushed

**Pull request merged**